components/NfcShare.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div v-if="supportsNfc">
+    <button
+      v-if="!isSharing"
+      @click="writeNfc">
+      Share via NFC
+    </button>
+    <div v-if="isSharing">
+      Hold your device near a reader to share your credential.
+      <button @click="cancelWrite">
+        Cancel
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+import {defineComponent, ref} from 'vue';
+import {Notify} from 'quasar';
+
+export default defineComponent({
+  props: {
+    credential: {
+      type: Object,
+      required: true
+    }
+  },
+  setup(props) {
+    const lenMaxNfcBytes = 32768;
+
+    const supportsNfc = ref(false);
+    const isSharing = ref(false);
+
+    let abortController = null;
+    let ndef = null;
+
+    if('NDEFReader' in window) {
+      supportsNfc.value = true;
+      // eslint-disable-next-line no-undef
+      ndef = new NDEFReader();
+    }
+
+    function cancelWrite() {
+      if(abortController) {
+        abortController.abort();
+      }
+      isSharing.value = false;
+    }
+
+    function notifyError(message) {
+      Notify.create({
+        message,
+        color: 'red-5',
+        timeout: 5000,
+        textColor: 'white',
+        actions: [{
+          label: 'Dismiss',
+          color: 'white'
+        }]
+      });
+    }
+
+    function notifySuccess(message) {
+      Notify.create({
+        message,
+        color: 'green-5',
+        timeout: 5000,
+        textColor: 'white',
+        actions: [{
+          label: 'Dismiss',
+          color: 'white'
+        }]
+      });
+    }
+
+    async function writeNfc() {
```

Comment:
I think we should lift this part out and make it part of bedrock-web-wallet instead because it's not specific to an UI framework, etc. -- but that can happen later after we get more experience and figure out a good API for working with it that wy. I'll turn this comment into an issue.